content/id/docs/concepts/configuration/secret.md

```diff
@@ -0,0 +1,1062 @@
+sebelum menjalankan perintah `kubectl apply`.
+
+stringData merupakan _field_ yang sifatnya _write-only_ untuk alasan kenyamanan pengguna.
+_Field_ ini tidak pernah ditampilkan ketika Secret dibaca. Sebagai contoh, misalkan saja kamu menjalankan
+perintah sebagai berikut:
+
+```shell
+kubectl get secret mysecret -o yaml
+```
+
+Keluaran yang diberikan kurang lebih akan ditampilkan sebagai berikut:
+
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  creationTimestamp: 2018-11-15T20:40:59Z
+  name: mysecret
+  namespace: default
+  resourceVersion: "7225"
+  selfLink: /api/v1/namespaces/default/secrets/mysecret
+  uid: c280ad2e-e916-11e8-98f2-025000000001
+type: Opaque
+data:
+  config.yaml: YXBpVXJsOiAiaHR0cHM6Ly9teS5hcGkuY29tL2FwaS92MSIKdXNlcm5hbWU6IHt7dXNlcm5hbWV9fQpwYXNzd29yZDoge3twYXNzd29yZH19
+```
+
+Jika sebuah _field_ dispesifikasikan dalam bentuk data maupun stringData,
+maka nilai dari stringData-lah yang akan digunakan. Sebagai contoh, misalkan saja terdapat
+definisi Secret sebagai berikut:
+
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: mysecret
+type: Opaque
+data:
+  username: YWRtaW4=
+stringData:
+  username: administrator
+```
+
+Akan menghasilkan Secret sebagai berikut:
+
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  creationTimestamp: 2018-11-15T20:46:46Z
+  name: mysecret
+  namespace: default
+  resourceVersion: "7579"
+  selfLink: /api/v1/namespaces/default/secrets/mysecret
+  uid: 91460ecb-e917-11e8-98f2-025000000001
+type: Opaque
+data:
+  username: YWRtaW5pc3RyYXRvcg==
+```
+
+Dimana string `YWRtaW5pc3RyYXRvcg==` akan di-_decode_ sebagai `administrator`.
+
+_Key_ dari data dan stringData yang boleh tersusun atas karakter alfanumerik,
+'-', '_' or '.'.
+
+**Catatan _Encoding_:** _Value_ dari JSON dan YAML yang sudah diseriakisasi dari data Secret
+akan di-_encode_ ke dalam string base64. _Newline_ dianggap tidak valid pada string ini dan harus
+dihilangkan. Ketika pengguna Darwin/macOS menggunakan alat `base64`, maka pengguna
+tersebut harus menghindari opsi `-b` yang digunakan untuk memecah baris yang terlalu panjang.
+Sebaliknya pengguna Linux _harus_ menambahkan opsi `-w 0` pada perintah `base64` atau
+melakukan mekanisme _pipeline_ `base64 | tr -d '\n'` jika tidak terdapat opsi `-w`.
+
+#### Membuat Secret dengan Menggunakan _Generator_
+Kubectl mendukung [mekanisme manajemen objek dengan menggunakan Kustomize](/docs/tasks/manage-kubernetes-objects/kustomization/)
+sejak versi 1.14. Dengan fitur baru ini, kamu juga dapat membuat sebuah Secret dari sebuah _generator_ 
+dan kemudian mengaplikasikannya untuk membuat sebuah objek pada Apiserver. _Generator_ yang digunakan haruslah 
+dispesifikasikan di dalam sebuah _file_ `kustomization.yaml` di dalam sebuah direktori.
+
+Sebagai contoh, untuk menghasilan sebuah Secret dari _file-file_ `./username.txt` dan `./password.txt`
+```shell
+# Create a kustomization.yaml file with SecretGenerator
```

Comment:
Should this comment be localized into Bahasa Indonesia?

Reply:
@sftim Have difficulties on the `glossary_tooltip` doesn't rendered properly even though I already created all needed glossaries